cli/module_build.go

```diff
@@ -727,74 +726,141 @@ func (c *viamClient) ensureModuleRegisteredInCloud(ctx *cli.Context, moduleID mo
 	return nil
 }
 
-// moduleCloudReload triggers a cloud build and then reloads the specified module with that build.
-func (c *viamClient) moduleCloudReload(ctx *cli.Context, args reloadModuleArgs, platform string) (string, error) {
-	manifest, err := loadManifest(args.Module)
+func (c *viamClient) inferOrgIDFromManifest(manifest moduleManifest) (string, error) {
+	moduleID, err := parseModuleID(manifest.ModuleID)
 	if err != nil {
 		return "", err
 	}
-
-	// ensure that the module has been registered in the cloud
-	moduleID, err := parseModuleID(manifest.ModuleID)
+	org, err := getOrgByModuleIDPrefix(c, moduleID.prefix)
 	if err != nil {
 		return "", err
 	}
 
-	err = c.ensureModuleRegisteredInCloud(ctx, moduleID, &manifest)
+	return org.GetId(), nil
+}
+
+func (c *viamClient) triggerCloudReload(
```

Comment:
nit: I read this initially and expected the entire flow for reload here. Then realized it was just the build trigger.
```suggestion
func (c *viamClient) triggerCloudReloadBuild(
```